eta/core/storage.py

```diff
@@ -1879,35 +1743,24 @@ def load_credentials(cls, credentials_path=None):
                 `google.auth.credentials.Credentials` instance and the path
                 from which it was loaded
```

Comment:
I would note that the tuple values may not be defined here `(None, None)`. I think the doc strings may need it too